src/main/webapp/js/instructorFeedbacks.es6

```diff
@@ -64,6 +65,18 @@ function checkFeedbackQuestion(form) {
         setStatusMessageToForm(DISPLAY_FEEDBACK_QUESTION_NUMSCALE_INTERVALINVALID, StatusType.DANGER, form);
         return false;
     }
+    if ($(form).find(`[name=${FEEDBACK_QUESTION_TYPE}]`).val() === 'RUBRIC') {
+        const questionNum = getQuestionNumFromEditForm(form);
+        const questionId = `#rubricNumRows-${questionNum}`;
+        const numberOfRows = parseInt($(questionId).val(), 10);
+        for (let row = 0; row < numberOfRows; row += 1) {
+            if ($.trim($(form).find(`[name=${FEEDBACK_QUESTION_RUBRIC_SUBQUESTION}-${row}]`).val()) === '' &&
+                $(form).find(`[name=${FEEDBACK_QUESTION_RUBRIC_SUBQUESTION}-${row}]`).get(0) !== undefined) {
```

Comment:
The variable ``$(form).find(`[name=${FEEDBACK_QUESTION_RUBRIC_SUBQUESTION}-${row}]`)`` can be extracted since it's used more than once. This would also make the condition more readable.